commons/zenoh-protocol/src/transport/join.rs

```diff
@@ -125,6 +129,13 @@ pub mod ext {
     /// # Shm extension
     /// Used to advertise shared memory capabilities
     pub type Shm = zextzbuf!(0x2, true);
+
+    /// # Patch extension
+    /// Used to negotiate the patch version of the protocol
+    /// if not present (or 0), then protocol as released with 1.0.0
+    /// if >= 1, then fragmentation start/stop marker
+    pub type Patch = zextz64!(0x3, false);
```

Comment:
Even if not necessary, it may be good to use the same extension ID of the `Init/Open`?

```suggestion
    pub type Patch = zextz64!(0x7, false);
```